src/model.py

```diff
@@ -229,6 +233,12 @@ def init_engine(self):
                 else:
                     raise e
 
+    def _get_bool_config_param(self, param_name: str) -> bool:
+        return (param_name in self.model_config["parameters"]) and (
+            self.model_config["parameters"][param_name]["string_value"].lower()
+            in ["yes", "true"]
```

Comment:
nit: make sure we don't actually use "yes" in our example.. We can parse it but we should use "true/false" ("enable/disable"?) for consistency

Reply:
I am more than happy to allow only "true", but "yes" is a legacy from the [metrics implementation](https://github.com/triton-inference-server/vllm_backend/blob/r24.11/src/model.py#L167-L168).

I think there are two ways to resolve it:
1. Have the metrics parse the bool value differently than the health check, but this will introduce inconsistency within the vLLM backend.
2. Update the metrics bool parsing to accept "true" only, but this can be a breaking change for existing users.

Reply:
as its a "bool config param" suggest "true/false" only

Reply:
I guess I would be in favor of breaking compat - to my knowledge we don't use 'yes' anywhere else ....

Reply:
sure, updated bool config param to accept true/false only
https://github.com/triton-inference-server/vllm_backend/pull/73/commits/a26a083755da584ce3b117e09c907d4b0f90c71c